config/locales/doc_auth/es.yml

```diff
@@ -197,7 +197,7 @@ es:
       document_capture_subheader_selfie: Fotografía de su cara
       document_capture_with_selfie: Agregue las fotos de su identificación y una foto de usted
       front: Anverso de su licencia de conducir o identificación estatal
-      how_to_verify: Elija cómo quiere verificar su identidad
+      how_to_verify: Elija cómo desea verificar su identidad.
```

Comment:
The [English and French translation](https://docs.google.com/document/d/1vhGmTiF312PjOPWYvJH9oV0X95r7OIgGnQKLDCYGeVI/edit#heading=h.8uogezwgll7m) do not contain a period. Nor does the code. I bet the Spanish translation should no. I would delete it.